### DISCUSSION TITLE
Some routes are categorized incorrectly in the [documentation](https://docs.rsshub.app/routes).

### DISCUSSION THREAD

**Discussion by howfool** [DIYgod/RSSHub#16800] [Q&A] [answered]
Hello maintainers, I found that seems all the routes placed in `routes/gov` directory are marked as *上海市人民政府* on the [documentation page](https://docs.rsshub.app/routes/government). In fact, **only** routes in `routes/gov/sh` directory should be marked as *上海市人民政府*.

The thing that seems to happen is that during `build:docs`, the whole `routes/gov` dir is given a same namespace `gov` after ‘registry.ts’ was run. The `name` and `url` fields are repeatedly overwritten, when each time `namespace.ts` is read. And `sh` is the last subfolder containing `namespace.ts` in alphabetical order.

In my opinion this seems to be some kind of bug?

Kind regards.

**Comment by TonyRL** [comment, marked as answer]
Yes, many of the `/gov` routes are glitched because the migration from hard-coding route entries https://github.com/DIYgod/RSSHub/blob/9d980d767089389267ba9d319a68de44954ce9a0/lib/v2/aqara/router.js#L2
to the newer route registration #14718 is done by script which is a hit-and-miss. To fix that, it is required to add those missing `namespace.ts` and a slight tweak of `scripts/workflow/build-docs.ts`
